tutorial-content/content/weighting-schemes.md

```diff
@@ -267,6 +298,19 @@ Now, note that the edge of the PRISM data is in the middle of the LandScan grid
 landscan <- disaggregate(landscan, fact=2) / 4
 ```
 ````
+````{tab-item} Python
+```Python
+import numpy as np
+
+# Disaggregate
+fact = 2
+landscan_disaggregated = np.repeat(np.repeat(landscan, fact, axis=0), fact, axis=1) / (fact**2)
+
+# Optionally, you can save the disaggregated raster to a new file
+landscan = rasterio.Affine(src.transform.a / fact, src.transform.b, src.transform.c,
```

Comment:
`src` isn't defined here, and I think this line just creates the affine transformation but doesn't actually apply it. I think you should just drop this line.

Reply:
Ok